src/content/en/fundamentals/performance/optimizing-javascript/code-splitting/index.md

```diff
@@ -0,0 +1,745 @@
+chunk.
+
+Preloading, in my opinion, makes the most sense for scripts critical to
+rendering the initial route. Twitter does this to speed up loading of the
+[Twitter Lite](https://mobile.twitter.com/home) app:
+
+<figure>
+  <img src="images/figure-9-1x.png" srcset="images/figure-9-1x.png 1x,
+images/figure-9-2x.png 2x" alt="Twitter Lite DOM snapshot in DevTools revealing
+several preloaded JavaScript resources.">
+  <figcaption><b>Figure 9</b>. Twitter Lite DOM snapshot in DevTools revealing
+several preloaded JavaScript resources.</figcaption>
+</figure>
+
+Unfortunately, `webpackPreload` only works with dynamic `import()` calls, so in
+order to preload chunks critical to the initial route in the example app, we
+need to rely on another method involving a useful plugin called
+[preload-webpack-plugin](https://github.com/GoogleChromeLabs/preload-webpack-plugin).
+Once this plugin is installed, we bring it into the webpack config like so:
+
+```javascript
+const PreloadWebpackPlugin = require("preload-webpack-plugin");
+```
+
+Then we configure the plugin to preload the `main` and `vendors` chunks by
+adding an instance of the plugin to the `plugins` array:
+
+```javascript
+plugins: [
+  // Other plugins omitted...
+  new PreloadWebpackPlugin({
+    rel: "preload",
+    include: ["main", "vendors"]
+  })
+]
+```
+
+This configuration will place add preload hints via `<link>` elements in the
+`<head>` for both the `vendors` and `main` chunks.
+
+<figure>
+  <img src="images/figure-10-1x.png" srcset="images/figure-10-1x.png 1x,
+images/figure-10-2x.png 2x" alt="Preload hints added to the &lt;head&gt; of the
+document for the main and vendors chunks as seen in DevTools.">
+  <figcaption><b>Figure 10</b>. Preload hints added to the
+<code>&lt;head&gt;</code> of the document for the <code>main</code> and
+<code>vendors</code> chunks as seen in DevTools.</figcaption>
+</figure>
+
+While this doesn't confer much of a performance boost in the example app, it
+_can_ boost loading performance in apps where there are many chunks of
+JavaScript and other resources that would otherwise contend for bandwidth. To
+see preloading in action in the example app, check out the
+[webpack-dynamic-splitting-preload
+branch](https://github.com/malchata/code-splitting-example/tree/webpack-dynamic-splitting-preload).
+
+Note: preload-webpack-plugin _must_ be used with html-webpack-plugin! When
+adding it to your `plugins` array, be sure to place it after the last instance
+of html-webpack-plugin.
+
+## Conclusion and resources
+
+There's no doubt that code splitting is hard to do. What's more, _how_ you'll
+split code in your specific app will take time for you to suss out. If you're
+curious to know more about code splitting, or want to read some different takes
+on the subject, check out this list of articles:
+
+- [Official Parcel.js code splitting
+docs.](https://parceljs.org/code_splitting.html)
+- [Official webpack code splitting
```

Comment:
Can we flip the order here so Webpack's code-splitting guide is first, followed by Parcel? (just going off of tool usage/popularity)